src/elexmodel/handlers/data/CombinedData.py

```diff
@@ -163,24 +190,74 @@ def _get_unexpected_units(self, aggregates):
 
         return unexpected_units
 
-    def _get_non_modeled_units(self, percent_reporting_threshold, turnout_factor_lower, turnout_factor_upper):
-        expected_geographic_units = self._get_expected_geographic_unit_fips().tolist()
+    def _fit_outlier_detection_model(self, reporting_units, response_variable, outlier_z_threshold):
+        features = ["baseline_normalized_margin", "ethnicity_likely_african_american", "percent_bachelor_or_higher"]
+        features_to_use = [feature for feature in features if feature in reporting_units.columns]
```

Comment:
I know we'll always have `postal_code` and `baseline_normalized_margin` available as features, but what about the other two?  Or what if they're named `race_african_american` or `education_bachelors_or_higher`? 🤔

Reply:
I added those also! now we have four there and we basically just pick the relevant ones that are present in the reporting units pd in the next line

Reply:
Great!  And also thanks for getting the name of the feature right compared to what I wrote lol 😄 🎉